lib/carrierwave/storage/fog.rb

```diff
@@ -273,7 +273,7 @@ def initialize(uploader, base, path)
         # === Returns
         #
         # [String] contents of file
-        def read
+        def read(bytes = nil)
           file.body
```

Comment:
When I tried it it seemed to be fine to return the entire body. I couldn't figure out a way of returning chunks in fog